docker/Makefile

```diff
@@ -41,11 +44,21 @@ ifdef no_proxy
 endif
 ifdef DO_PROXY
    DOCKER_COMPOSE_FILES += sawtooth-pdo.proxy.yaml
+   DOCKER_COMPOSE_FILES_CCF += ccf-pdo.proxy.yaml
 endif
 ifdef PDO_DEBUG_BUILD
    DOCKER_COMPOSE_FILES += sawtooth-pdo.debugging.yaml
 endif
+ifdef PDO_LEDGER_TYPE # not sure how to pull this check as part of the test-ccf target, so that we do not need the extra env PDO_LEDGER_TYPE
+ifeq ($(PDO_LEDGER_TYPE),ccf)
+ifndef host_ip
+$(error Incomplete configuration, define env host_ip to test pdo/ccf via docker)
```

Comment:
see the update for PDO_HOSTNAME which may help address this. the configuration files should reliably expand the hostname into a viable address. you can also get the host ip in an env variable (from a name) with 
export host_ip="$(dig +short ${PDO_HOSTNAME})"

Reply:
Thank you both, the entire section of code is now removed. host_ip need not be set anymore like before. CCF container will use compose default networking rather than host networking. pdo container will set ledger url as http://pdo-tp-ccf:6600.